temporian/core/operators/drop_index.py

```diff
@@ -137,35 +143,37 @@ def build_op_definition(cls) -> pb.OperatorDef:
 operator_lib.register_operator(DropIndexOperator)
 
 
-def _process_index_names(
+def _normalize_index_to_drop(
     event: Event, index_names: Optional[Union[List[str], str]]
 ) -> List[str]:
     if index_names is None:
-        return event.sampling.index
+        return event.sampling.index.names
 
     if isinstance(index_names, str):
         index_names = [index_names]
 
     if len(index_names) == 0:
         raise ValueError("Cannot specify empty list as `index_names` argument.")
 
+    # TODO: Don't recompute name each time.
     # check if any index names are missing from the index
     missing_index_names = [
         index_name
         for index_name in index_names
-        if index_name not in event.sampling.index
+        if index_name not in event.sampling.index.names
     ]
     if missing_index_names:
         raise KeyError(
-            f"{missing_index_names} are missing from the input event's index."
+            f"Dropped indexes {missing_index_names} are missing from the"
+            f" input index. The input index is {event.sampling.index}."
```

Comment:
`{event.sampling.index} -> {event.sampling.index.names}` is more readable for this operator imo

Reply:
*Done